src/design-system/InputTextArea.tsx

```diff
@@ -1,7 +1,8 @@
-import React from "react";
+import React, { useEffect, useRef } from "react";
```

Comment:
This component wasn't being used anywhere else so I updated it to match the style and functionality of our other `Input` components.